src/py42/sdk/__init__.py

```diff
@@ -39,8 +40,8 @@
 
 def from_local_account(host_address, username, password, totp=None):
     """Creates a :class:`~py42.sdk.SDKClient` object for accessing the Code42 REST APIs using the
-    supplied credentials. Currently, only accounts created within the Code42 console or using the
-    APIs (including py42) are supported. Username/passwords that are based on Active Directory,
+    supplied credentials. This method supports only accounts created within the Code42 console or using the
```

Comment:
@annie-payseur  Now that user can use any custom authentication, this and similar documentation `docs\methods.md` might need improvement.

Reply:
Thanks for letting me know, @kiran-chaudhary!